README.md

```diff
@@ -1,31 +1,72 @@
-# Simple Pelion Device Management Client
+# Mbed Device Management
 
 (aka Simple Pelion DM Client, SPDMC, previously called Simple Mbed Cloud Client)
 
```

Comment:
Maybe you can get rid of the aka Simple Pelion DM .....)?
IMO it shouldn't be shown, unless you point at the repos with all the history...